src/constants/index.ts

```diff
@@ -0,0 +1,26 @@
+const INITIAL_DATA_LOAD_COUNT = 20;
+const SUBSEQUENT_DATA_LOAD_COUNT = 4;
+const MAX_CART_ITEMS_COUNTS = 100;
+
+const CATEGORY: Record<string, string> = {
```

Comment:
as const 로 타입단언 해주셨으니, 따로 타입선언은 할 필요 없어보여요 ㅎㅎ
```suggestion
const CATEGORY = {
```

Reply:
넵 수정했습니다. 타입 단언을 사용하니 Record<string, string>가 필요없었네요